taskforce-services/taskforce-next-admin
# fixes #1 Settings page

# Settings pipedream secret keys

Closes 

## Description
This page have a text box that allows the user to enter their API keys. When the user hits the "Save" button, add the keys to the browser localstorage. Then, return the API keys. 

## Screenshot
<img width="1440" alt="Screen Shot 2022-07-21 at 14 24 14" src="https://user-images.githubusercontent.com/79850230/180275842-49de2753-b808-4972-8a1c-b6cf1aa9ef01.png">
<img width="1440" alt="Screen Shot 2022-07-21 at 14 24 25" src="https://user-images.githubusercontent.com/79850230/180275852-5379a451-b0ad-47f7-81b0-7d6557697ba7.png">
<img width="1440" alt="Screen Shot 2022-07-21 at 14 24 31" src="https://user-images.githubusercontent.com/79850230/180275857-ef5814de-bf87-4d9a-bcc8-82135ea77f78.png">
<img width="1434" alt="Screen Shot 2022-07-21 at 14 24 52" src="https://user-images.githubusercontent.com/79850230/180275858-3964d87f-1793-4370-a646-bb01e043d1cc.png">

## Steps to Verify
Enter a valid secret key and you should see the keys returned
Enter an invalid secret key and you should see an error message
